.gitignore

```diff
@@ -27,7 +27,7 @@ hs_err_pid*
 
 # Ignore Gradle build output directory
 build
-
+src/main/java/jalau/at18/katas/langtonan
```

Comment:
Hi @Git-Tonio , we should not ignore these files as they are part of the project. I see you have commits in your branch that shouldn't be, maybe that's the reason this was added? In such case, your branch should have been created from main so it does not pull other commits unrelated.

You can re-create your branch to solve this, unfortunately, we cannot merge this change because it would hide the files under langton ant kata